### PR TITLE
Submits embargo release date when depositing.

### PR DESCRIPTION
**Pull request by justinlittman**
refs https://github.com/sul-dlss/sdr-api/issues/122

## Why was this change made?
So that the embargo release date will be provided when depositing.

## Was the documentation (README, DevOpsDocs, wiki) updated?
No.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ gem 'nokogiri'
 gem 'okcomputer'
 gem 'pg'
 gem 'rake'
-gem 'sdr-client', '~> 0.9.0'
+gem 'sdr-client', '~> 0.10'
```
Comment (mjgiarlo) [inline Gemfile]: We've been pinning this to the patch-level to be conservative re: breaking changes. I don't care strongly about changing this. Maybe @jcoyne does?

File: app/services/file_metadata_builder.rb [hunk 2]
```diff
@@ -34,7 +34,7 @@ def build
         OPERATIONS.each do |operation|
           result = operation.for(file_path: file_path,
                                  embargo_until: @embargo_until)
-          next unless result
+          next if result.nil?
```
Comment (mjgiarlo) [inline app/services/file_metadata_builder.rb]: Prefer this more explicit approach.

Comment (justinlittman) [reply app/services/file_metadata_builder.rb]: @mjgiarlo Not clear from your comment if you prefer original or change. Reason for the change is that if result was `false` it nexting.

Comment (mjgiarlo) [reply app/services/file_metadata_builder.rb]: Oh, I prefer the change you made.

**Pull request merged**